Build/phpmd.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<ruleset name="phpList">
+    <description>
+        PHPMD rules for Tea
+    </description>
+
+    <!-- The commented-out rules will be enabled once the code does not generate any warnings anymore. -->
+
+    <rule ref="rulesets/cleancode.xml/BooleanArgumentFlag"/>
+    <rule ref="rulesets/cleancode.xml/StaticAccess"/>
```

Comment:
We need to drop this rule as we often have static access in TYPO3 extensions, e.g., `GeneralUtility::makeInstance`.